packages/legend-query/src/components/QueryBuilderResultPanel.tsx

```diff
@@ -103,14 +111,38 @@ export const QueryBuilderResultPanel = observer(
     const { queryBuilderState } = props;
     const applicationStore = useApplicationStore();
     const resultState = queryBuilderState.resultState;
+    const queryParametersState = queryBuilderState.queryParametersState;
     const executionResult = resultState.executionResult;
-    const execute = (): void => {
-      if (queryBuilderState.queryParametersState.parameters.length) {
-        queryBuilderState.queryParametersState.setValuesEditorIsOpen(true);
+    const _execute = (): Promise<void> =>
```

Comment:
why do we split it out of `execute()`? and why do we need `_exportQuery()`? Let's flatten them